modeling_tool.cc

```diff
@@ -34,40 +34,62 @@ void ReadWriteTransaction(google::cloud::spanner::Client client) {
   namespace spanner = ::google::cloud::spanner;
   using ::google::cloud::StatusOr;
 
-  // A helper to read a single album MarketingBudget.
-  auto get_current_age =
+  // A helper to read a single Singers DefaultAge.
+  auto get_default_age =
       [](spanner::Client client, spanner::Transaction txn,
          std::int64_t id) -> StatusOr<std::int64_t> {
     auto key = spanner::KeySet().AddKey(spanner::MakeKey(id));
     auto rows = client.Read(std::move(txn), "Singers", std::move(key),
                             {"DefaultAge"});
-    
-    //auto rows = client.Read(std::move(txn), "Singers", spanner::KeySet::All(),
-      //                      {"DefaultAge"});
+    using RowType = std::tuple<std::int64_t>;
+    auto row = spanner::GetSingularRow(spanner::StreamOf<RowType>(rows));
+    if (!row) return std::move(row).status();
```

Comment:
why move is needed here?

Reply:
this is copied from example code, i was guessing row is a l-value reference type, and calling move here can invoke move constructor in the return statement instead of copy constructor, thus reduce the cost